packages/actor-abstract-mediatyped/lib/ActorAbstractMediaTyped.ts

```diff
@@ -46,32 +46,36 @@ export abstract class ActorAbstractMediaTyped<HI, HT, HO>
    *
    * @param {HI} action The handle action to test.
    * @param {string} mediaType The media type to test.
+   * @param {ActionContext} context An optional context.
    * @return {Promise<T>} A promise that resolves to the handle test result.
    */
-  public abstract async testHandle(action: HI, mediaType: string): Promise<HT>;
+  public abstract async testHandle(action: HI, mediaType: string, context?: ActionContext): Promise<HT>;
```

Comment:
Should the context parameter be optional? This might encourage developers to not pass on the context when they receive it.

Reply:
Yes, that makes sense. The context can still be null, but all remove the `?`everywhere to make it clear that the context must be passed along.